org.srb2.SRB2.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: org.srb2.SRB2
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: srb2
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --persist=.srb2
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: game-music-emu
+    buildsystem: cmake-ninja
+    cleanup:
+      - /include
+      - /lib/*.so
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.3.tar.xz
+        sha256: aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d
+
+  - name: libopenmpt
+    config-opts:
+      - --disable-examples
+      - --disable-openmpt123
+      - --disable-static
+      - --disable-tests
+      - --without-portaudio
+      - --without-portaudiocpp
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.so
+      - /lib/pkgconfig
+      - /share/doc
+    sources:
+      - type: archive
+        url: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-0.4.12+release.autotools.tar.gz
+        sha256: 0ccd64476e6c8a084277e7093c4034d702e7999eeffd31adc89b33685e725e60
+
+  - name: srb2
+    buildsystem: simple
+    build-commands:
+      - 'make -C src -j $FLATPAK_BUILDER_N_JOBS LINUX64=1 NOUPX=1 SDL=1'
+      - 'install -D -m 755 -t $FLATPAK_DEST/bin bin/Linux64/Release/lsdl2srb2'
+      - 'install -D -m 644 srb2.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png'
+      - 'install -D -m 755 srb2.sh $FLATPAK_DEST/bin/srb2'
+      - 'install -D -m 644 -t $FLATPAK_DEST/share/applications $FLATPAK_ID.desktop'
+      - 'install -D -m 644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml'
+    sources:
+      - type: git
+        url: https://github.com/STJr/SRB2
+        tag: SRB2_release_2.2.4
+        commit: feced5ec3c469b2842c70e50663373dcd2d0cd2f
+      - type: script
+        commands:
+          - 'ls $HOME/.srb2/srb2.pk3 || zenity --error --text "Game assets not found, please add them to your ~/.srb2 folder. See " --ok-label "Close" --width=400'
```

Comment:
Also this path is wrong. It should be `~/.var/app/org.srb2.SRB2/.srb2`.

Reply:
You mean the path tested, the path in the error dialog or both? My understanding was that `~./.srb2` would be mapped to `~/.var/app/org.srb2.SRB2/.srb2` and `$HOME` would point to `~/.var/app/org.srb2.SRB2`?

Reply:
`--persist` causes given directory to be symlinked from `~/.var/app/appid/.foo` to `~/.foo` but only inside the sandbox. The host `~/.foo` is not involved at all.

Reply:
I see, I thought it was the over way around. I know what to do now :+1:

Reply:
It should be fixed now.